mesos-client/src/main/scala/com/mesosphere/mesos/client/Session.scala

```diff
@@ -61,7 +58,10 @@ case class Session(baseUri: Uri, streamId: String, authorization: Option[Credent
     import system.dispatcher
     logger.info(s"Create authenticated session for stream $streamId.")
     val sessionActor =
-      system.actorOf(SessionActor.props(authorization, streamId, baseUri.withPath(Path("/api/v1/scheduler"))))
+      system.spawn(
+        SessionActor(authorization, streamId, baseUri.withPath(Path("/api/v1/scheduler"))),
+        s"SessionActor-$streamId")
```

Comment:
This must be unique. So it'll crash in case we use two session actors for the same stream. Which I think is actually what we want.